packages/core/src/server/index.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/ban-ts-ignore, require-atomic-updates */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
@typescript-eslint/ban-ts-ignore was replaced by ban-ts-comment in https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0